src/Arcus.WebApi.Logging.Core/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -74,6 +75,23 @@ public static IServiceCollection AddHttpCorrelation(this IServiceCollection serv
         {
             Guard.NotNull(services, nameof(services), "Requires a services collection to add the HTTP correlation services");
 
+            return AddHttpCorrelation(services, configureOptions, configureMicrosoftApplicationInsightsOptions: null);
+        }
+
+        /// <summary>
+        /// Adds operation and transaction correlation to the application.
+        /// </summary>
+        /// <param name="services">The services collection containing the dependency injection services.</param>
+        /// <param name="configureOptions">The function to configure additional options how the correlation works.</param>
+        /// <param name="configureMicrosoftApplicationInsightsOptions"></param>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="services"/> is <c>null</c>.</exception>
+        public static IServiceCollection AddHttpCorrelation(
```

Comment:
Actually, we don't need this overload as the configuration can be changed afterwards. Will change that.